src/main/java/somethingrandom/entity/ItemKind.java

```diff
@@ -0,0 +1,7 @@
+package somethingrandom.entity;
+
+public enum ItemKind {
```

Comment:
This strikes me as a magnet for open/closed violations, why do we need it? (Might be my fault for putting it in the design, sorry!)